components/page/list/text-search.vue

```diff
@@ -15,6 +15,7 @@
     const   eventBus   = useEventBus();
     const   queryText  = ref(route.query.freeText || '')
 
+    const disabled = ref(false);
```

Comment:
you added this but it's alwas false

Reply:
resolved [d313e9d](https://github.com/scbd/bioland-head/pull/44/commits/d313e9d6b65b4ed647479bfca50140c96ae00f7a)